Makefile.dryice.js

```diff
@@ -452,10 +457,10 @@ function optimizeTextModules(sources) {
         if (/\.css$/.test(pkg.id)) {
             // remove unnecessary whitespace from css
             input = input.replace(/\n\s+/g, "\n");
-            input = '"' + input.replace(/\r?\n/g, '\\\n') + '"';
+            input = '"' + input.replace(/\n/g, '\\\n') + '"';
```

Comment:
that will put whole string on one line, this keep it readable in non minified mode